### PR TITLE
makefile: add profiling support

### PR DESCRIPTION
**Pull request by safchain**

### CODE REVIEW

**Comment by nplanel** [comment]
LGTM

File: skydive_prof.go [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+	for {
+		<-c
+		f, err := os.Create("/tmp/skydive-cpu.prof")
+		if err != nil {
+			log.Fatal("could not create CPU profile: ", err)
+		}
+		if err := pprof.StartCPUProfile(f); err != nil {
+			log.Fatal("could not start CPU profile: ", err)
+		}
+		<-c
```
Comment (nplanel) [inline skydive_prof.go]: may add log.Info("CPU profiling Started")

File: skydive_prof.go [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+		pprof.StopCPUProfile()
+	}
```
Comment (nplanel) [inline skydive_prof.go]: log.Info("CPU profiling Stopped")

File: skydive_prof.go [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+}
+
+func main() {
+
+	c := make(chan os.Signal, 1)
+	signal.Notify(c, syscall.SIGUSR1)
+	go profile(c)
+
+	defer cmd.RootCmd.Execute()
```
Comment (nplanel) [inline skydive_prof.go]: you love defer

**Comment by safchain** [comment]
rerun functional-tests-backend-orientdb

**Pull request merged**